src/i18n/locales/en/translation.json

```diff
@@ -334,6 +334,19 @@
           "connection_failed": "Connection failed. Are your credentials correct?"
         }
       },
+      "albyhub": {
+        "title": "Alby Hub",
+        "page": {
+          "instructions": "Paste a NWC connection string from Alby Hub",
```

Comment:
It's called "connection secret", please also adjust existing items.

Reply:
resolved